.github/workflows/ci.yml

```diff
@@ -162,7 +162,7 @@ jobs:
           node-version: ${{ matrix.node_version_to_setup }}
 
       - name: Install Dependencies
-        run: npm ci --ignore-scripts
+        run: npm install --ignore-scripts
```

Comment:
We can't do that, CI should use `npm ci` I'm working on a fix.
It will remove internal hack that we use to have our own ESLint plugin

Reply:
more details: `npm install` is bad because it changes lock file